build.rs

```diff
@@ -1,14 +1,17 @@
-use cfgrammar::yacc::YaccKind;
+use cfgrammar::yacc::{YaccKind, YaccOriginalActionKind};
 use lrlex::CTLexerBuilder;
 
-fn main() -> Result<(), Box<dyn std::error::Error>> {
+fn main() {
     CTLexerBuilder::new()
+        .rust_edition(lrlex::RustEdition::Rust2021)
         .lrpar_config(|ctp| {
-            ctp.yacckind(YaccKind::Grmtools)
+            ctp.yacckind(YaccKind::Original(YaccOriginalActionKind::GenericParseTree))
```

Comment:
Aha, this is a rare use of `GenericParseTree` in the wild :) I suggest we let this PR through, but the next PR should, IMHO, use actions to make an AST on the fly (see e.g. https://github.com/ltratt/pizauth/blob/master/src/config.y). Generic parse trees are, IMHO, a pain to work with.

Reply:
Added "actions" to TODO tasks in README https://github.com/softdevteam/pavel.yaiwr/pull/7/commits/bde317ac0f1e3c2ca1ab6cba5d7adfd554d62146